src/lesson8/classwork/table_creating_views.sql

```diff
@@ -0,0 +1,67 @@
+CREATE VIEW products_suppliers_categories AS
+SELECT product_name, quantity_per_unit, unit_price, units_in_stock, company_name,
+contact_name, phone, category_name, description
+FROM products
+JOIN suppliers USING(supplier_id)
+JOIN categories USING(category_id);
```

Comment:
much better to write like this:

SELECT 
  product_name, 
  quantity_per_unit, 
  unit_price, 
  units_in_stock, 
  company_name,
  contact_name, 
  phone, 
  category_name, 
  description
FROM products
JOIN suppliers USING(supplier_id)
JOIN categories USING(category_id);

Please, check style in other queries.